products/compute/api.yaml

```diff
@@ -17421,3 +17421,91 @@ objects:
+          api: 'https://cloud.google.com/compute/docs/reference/rest/v1/targetGrpcProxies'
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        kind: 'compute#operation'
+        path: 'name'
+        base_url: 'projects/{{project}}/global/operations/{{op_id}}'
+        wait_ms: 1000
+      result: !ruby/object:Api::OpAsync::Result
+        path: 'targetLink'
+      status: !ruby/object:Api::OpAsync::Status
+        path: 'status'
+        complete: 'DONE'
+        allowed:
+          - 'PENDING'
+          - 'RUNNING'
+          - 'DONE'
+      error: !ruby/object:Api::OpAsync::Error
+        path: 'error/errors'
+        message: 'message'
+    properties:
+      - !ruby/object:Api::Type::Time
+        name: 'creationTimestamp'
+        description: 'Creation timestamp in RFC3339 text format.'
+        output: true
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        description: |
+          Name of the resource. Provided by the client when the resource
+          is created. The name must be 1-63 characters long, and comply
+          with RFC1035. Specifically, the name must be 1-63 characters long
+          and match the regular expression [a-z]([-a-z0-9]*[a-z0-9])? which
+          means the first character must be a lowercase letter, and all
+          following characters must be a dash, lowercase letter, or digit,
+          except the last character, which cannot be a dash.
+        input: true
+        required: true
+      - !ruby/object:Api::Type::String
+        name: 'description'
+        description: 'An optional description of this resource.'
+        input: true
+      - !ruby/object:Api::Type::String
+        name: 'selfLinkWithId'
+        description: 'Server-defined URL with id for the resource.'
+        output: true
+      - !ruby/object:Api::Type::String
+        name: 'urlMap'
+        description: |
+          URL to the UrlMap resource that defines the mapping from URL to
+          the BackendService. The protocol field in the BackendService
+          must be set to GRPC.
+      - !ruby/object:Api::Type::Boolean
+        name: 'validateForProxyless'
+        description: |
+          If true, indicates that the BackendServices referenced by
+          the urlMap may be accessed by gRPC applications without using
+          a sidecar proxy. This will enable configuration checks on urlMap
+          and its referenced BackendServices to not allow unsupported features.
+          A gRPC application must use "xds:///" scheme in the target URI
+          of the service it is connecting to. If false, indicates that the
+          BackendServices referenced by the urlMap will be accessed by gRPC
+          applications via a sidecar proxy. In this case, a gRPC application
+          must not use "xds:///" scheme in the target URI of the service
+          it is connecting to
+      - !ruby/object:Api::Type::Fingerprint
+        name: 'fingerprint'
```

Comment:
Instead of defining a `fingerprint` field, I'd suggest updating `urlMap` as if it were a custom update. Doing that supports configuring a `fingerprint_value` where Terraform performs a GET immediately before doing an apply. See https://github.com/GoogleCloudPlatform/magic-modules/blob/master/products/compute/api.yaml#L13598-L13601